code/modules/mob/mob.dm

```diff
@@ -937,6 +937,13 @@ var/list/slot_equipment_priority = list( \
 
 	return 1
 
+mob/proc/isincrit()
+	return 0
+	
+mob/living/carbon/human/isincrit()
```

Comment:
Bad idea to leave procs referring to objects in unrelated files willy nilly. Should've said this in the discord, sorry. Put this proc in human_damage.dm or something relating to human helper procs and the mob/proc in mob_helpers.dm